storybook/src/stories/UseWindowListener/Resize.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { useWindowListener } from 'too-many-hooks'
+
+interface Props {
+  onResize: (name?: string | null) => string
+}
+
+const Resize: React.FC<Props> = ({ onResize }) => {
+  useWindowListener('resize', () =>
+    onResize(`Window resized to ${window.innerWidth} x ${window.innerHeight}`),
+  )
+  return (
+    <div className="prose">
+      Resize this window to trigger listener events, viewable in the &quot;Actions&quot; tab
+    </div>
```

Comment:
Text here is pretty small on desktop and also clashes in dark mode. Perhaps we should extract a re-usable text component that changes color depending on the theme and also scales with the container size.

<img width="435" alt="image" src="https://user-images.githubusercontent.com/10625031/215298895-3f98abba-1bb0-4868-9575-3e25bff397f5.png">

Reply:
Similar response to the other PR, I will make it bigger and I think #25 should cover the rest